src/llvm-late-gc-lowering.cpp

```diff
@@ -811,12 +804,13 @@ void LateLowerGCFrame::NoteUse(State &S, BBState &BBS, Value *V, BitVector &Uses
     }
 }
 
-void LateLowerGCFrame::NoteOperandUses(State &S, BBState &BBS, User &UI, BitVector &Uses) {
+void LateLowerGCFrame::NoteOperandUses(State &S, BBState &BBS, User &UI) {
     for (Use &U : UI.operands()) {
         Value *V = U;
-        if (!isSpecialPtr(V->getType()) && !isSpecialPtrVec(V->getType()))
-            continue;
-        NoteUse(S, BBS, V, Uses);
+        if (isSpecialPtr(V->getType()) || isSpecialPtrVec(V->getType()) ||
```

Comment:
I guess, this is the bugfix part of this commit? Add a test? I think `UpExposedUsesUnrooted` was left over from when we still used to optimize that. Also with test, I think this can be a separate PR, so we can get it in quicker.